api/handlers.go

```diff
@@ -251,6 +252,11 @@ func postImagesCreate(c *context, w http.ResponseWriter, r *http.Request) {
 	wf := NewWriteFlusher(w)
 
 	if image := r.Form.Get("fromImage"); image != "" { //pull
+		authConfig := dockerclient.AuthConfig{}
+		buf, err := base64.URLEncoding.DecodeString(r.Header.Get("X-Registry-Auth"))
```

Comment:
when `fromImage` is not provided it means import, and we don't need `authConfig` for import